### ISSUE TITLE
windows/app theme does not referesh the system after switching!

### ISSUE THREAD

**Issue by ghost** [t1m0thyj/WinDynamicDesktop#427]
- WDD Version: 5.1
- OS Version: 11 lastest

Steps to Reproduce:

1.change light to dark mode
2. pressing start menu shows whitetext on white surface+app that needs to be updated doesnot referesh unless refreshed manuallY!

Are you using the Microsoft Store version of WDD?: No

**Comment by t1m0thyj** [comment]
Can you check if the same behavior happens when you manually switch between light and dark themes in Windows 11 Personalization settings? If so, this is a bug in Windows rather than WDD.

**Comment by yazan1107** [comment]
we checked this me and the other app developer, the problem is from your side(maybe the scripts or something)

**Comment by snowtyler** [comment]
For me, this issue occurs on the latest Windows 11 Beta build (22593.1) with WDD 5.1. I tick "Enable Night Mode," apps such as browsers switch to dark as expected but the system UI and the background remains light.

**Comment by yazan1107** [comment]
here is a demonstration:
your app does not refresh the system theme properly so the app looks like this:
![wrongColor](https://user-images.githubusercontent.com/78726445/163590276-d3001e95-7b4a-443c-a12c-74f1c490a050.png)
after refresh(the supposed look of the app):
![rightColor](https://user-images.githubusercontent.com/78726445/163590323-9111352f-293b-4d97-93dc-8892842cc1ea.png)

**Comment by oreomaker** [comment]
The same problem have bothered me for some time.
I just found that when you change the SystemUsesLightTheme in the registry directly to false, the taskbar stay the same while the start menu and icons operate properly. I wonder what changes have been made when you change the color theme in the system settings.

**Comment by shust** [comment]
The problem is still relevant in the latest version of windows 22621.1105

**Comment by leostudiooo** [comment]
My system:

**Version**	Windows 11 Pro Workstation (_Insider Preview_)
**Version**	22H2
**Installation Date**	‎2023/‎1/‎22
**OS Version**	25281.1000
**Experience**	Windows Feature Experience Pack 1000.25281.1000.0

------

I checked the registry mentioned in the script. Things come strange.

Here's how it comes:
- If I change the color schedule manually in the `Settings App`, the registry value changes. Meanwhile the app windows and the taskbar changes color.
- If I click `refresh wallpaper` in WDD, the registry value changes to the same value as manually triggered. Meanwhile, the app windows changes its color, but the taskbar remains the same. After a restart it will sync the registry.

I guess that the `Settings App` has got some other ways to communicate with the taskbar, bypassing the registry.

**Comment by gaoyang** [comment]
Looking for a solution.

**Comment by bigplayer-ai** [comment]
I have the same issue, I think the font color should be called and changing registry is not enough. 
But I don't what should be called.
Check my mouse script, How I called system paramters to update the new mouse value.
```
$CSharpSig = @’
[DllImport("user32.dll", EntryPoint = "SystemParametersInfo")]
public static extern bool SystemParametersInfo(
             uint uiAction,
             uint uiParam,
             uint pvParam,
             uint fWinIni);
‘@
$CursorRefresh = Add-Type -MemberDefinition $CSharpSig -Name WinAPICall -Namespace SystemParamInfo –PassThru
$CursorRefresh::SystemParametersInfo(0x0057,0,$null,0)
```
Pull request for the mouse script: 
https://github.com/t1m0thyj/WDD-scripts/pull/31
Mouse script where I called system parameters to update their mouse values: 
https://github.com/t1m0thyj/WDD-scripts/blob/a0994e51bab41a0baeb568fa77ef0c4664958909/ChangeMouseScheme.ps1

**Comment by shust** [comment]
Заметил такую вещь, если отключить и включить обратно эффект прозрачности в настройках, то тема приходит в нужный вид. Возможно, это можно как-то добавить в скрипт автопереключения дневной/ночной темы?
![Untitled-1](https://user-images.githubusercontent.com/7703814/220056441-6fcb7a72-fda0-4156-86d2-60f9ed069d57.png)

**Comment by adoginspace** [comment]
Looks like the AutoDarkMode team had the same issue and managed to get it working by updating the colorization: https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/issues/567

**Comment by t1m0thyj** [comment]
It seems like Microsoft has made theme switching more difficult in Windows 11, so there is no longer a simple way to do it from a PowerShell script. I would suggest using AutoDarkMode alongside WinDynamicDesktop if you want both features - dynamic wallpaper and automatic dark theme.

**Comment by leostudiooo** [comment]
> It seems like Microsoft has made theme switching more difficult in Windows 11, so there is no longer a simple way to do it from a PowerShell script. I would suggest using AutoDarkMode alongside WinDynamicDesktop if you want both features - dynamic wallpaper and automatic dark theme.

This solution seems less *elegant* since one has to run one more app in the background for that. Maybe I can try to fix this when I have more spare time but that will be months later.

**Comment by bigplayer-ai** [comment]
If that’s the case I think WDD should have a built in options to change dark/light theme switch that uses the way mentioned. On 4 Mar 2023, at 6:13, Li Lingfeng ***@***.***> wrote:﻿

It seems like Microsoft has made theme switching more difficult in Windows 11, so there is no longer a simple way to do it from a PowerShell script. I would suggest using AutoDarkMode alongside WinDynamicDesktop if you want both features - dynamic wallpaper and automatic dark theme.

This solution seems less elegant since one has to run one more app in the background for that. Maybe I can try to fix this when I have more spare time but that will be months later.

**Comment by t1m0thyj** [comment]
Prior to WinDD v4, there were built-in options to change app/system theme. I removed them since the purpose of the core app is only to display dynamic wallpapers, and moved them into separate PowerShell scripts.

Contributions to the WDD-scripts repo are welcome to fix the behavior of the Change App Theme and Change System Theme scripts 🙂

**Comment by Spiritreader** [comment]
> If that’s the case I think WDD should have a built in options to change dark/light theme switch that uses the way mentioned

@bigplayer-ai Honestly maintaining a functioning theme switch is nightmare territory. It almost never works reliably and you're chasing behavior that often breaks the entire windows UI for no apparent reason. 

I would assume maintaining WDD by itself is already enough work as is (and so is ADM as a matter of fact) 

> This solution seems less elegant since one has to run one more app in the background for that. Maybe I can try to fix this when I have more spare time but that will be months late

@leostudiooo It's already possible to call ADM via CLI, perform the operations you desire and then shut it down again, so you get no background app. You can also hide the tray icon so you won't even know its running temporarily.
Yes you have to configure ADM a bit, but it should work. 

As far as elegance is concerned. Automatic theme switching is absolutely not approved by Microsoft and hacks to achieve said theme switching cannot be elegant by design. We're not even sure that ADM will work in the future.

**Comment by bigplayer-ai** [comment]
Can you explain more about ADM clis?On 9 Mar 2023, at 14:31, Sam ***@***.***> wrote:﻿

If that’s the case I think WDD should have a built in options to change dark/light theme switch that uses the way mentioned

@bigplayer-ai Honestly maintaining a functioning theme switch is nightmare territory. It almost never works reliably and you're chasing behavior that often breaks the entire windows UI for no apparent reason.
I would assume maintaining WDD by itself is already enough work as is (and so is ADM as a matter of fact)

This solution seems less elegant since one has to run one more app in the background for that. Maybe I can try to fix this when I have more spare time but that will be months late

@leostudiooo It's already possible to call ADM via CLI, perform the operations you desire and then shut it down again, so you get no background app. You can also hide the tray icon so you won't even know its running temporarily.
Yes you have to configure ADM a bit, but it should work.
As far as elegance is concerned. Automatic theme switching is absolutely not approved by Microsoft and hacks to achieve said theme switching cannot be elegant by design. We're not even sure that ADM will work in the future.

**Comment by bigplayer-ai** [comment]
> 
@Spiritreader 
Do you have any idea How can I change mouse scheme using ADM CLI while using WDD? I tried thinking of a solution. as they support this feature.

**Comment by Spiritreader** [comment]
> > 
> 
> @Spiritreader Do you have any idea How can I change mouse scheme using ADM CLI while using WDD? I tried thinking of a solution. as they support this feature.

Use windows theme mode, configure two themes with different mouse cursors, then set the "ignore background" flag.

**Comment by bigplayer-ai** [comment]
Oh Thanks I just needed to update my ADM.
Btw, I am not succeeding running this in powershell 7 script, What could be the reason?
I think my problem is purely syntax.
```c
AutoDarkModeShell.exe --force-light --and-launch-service
AutoDarkModeShell.exe --exit
```

**Comment by Spiritreader** [comment]
> Oh Thanks I just needed to update my ADM. Btw, I am not succeeding running this in powershell 7 script, What could be the reason? I think my problem is purely syntax.
> 
> ```c
> AutoDarkModeShell.exe --force-light --and-launch-service
> AutoDarkModeShell.exe --exit
> ```

I'm sadly not proficient in powershell, but yes, it's probably related to escaping arguments and using the correct call syntax I would assume.
Also make sure you include the full path to the AutoDarkModeShell executable if you haven't put the ADM directory in your path variables.

I would also recommend specifying a custom timeout here:
```
AutoDarkModeShell.exe --force-light 10 --and-launch-service
```

There was a bug in older versions (including current stable, so ensure you're using the latest beta) that caused problems if it was omitted when specifying the launch service argument.

**Comment by bigplayer-ai** [comment]
I found the correct syntax for powershell 7:
```c
$params = @("--light", "10", "--and-launch-service")
& "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" @params
$params2 = @("--exit")
& "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" @params2
```
What does the custom timeout do? Why should I use it?

**Comment by Spiritreader** [comment]
> I found the correct syntax for PowerShell 7:
> 
> ```powershell
> $params = @("--light", "10", "--and-launch-service")
> & "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" @params
> $params2 = @("--exit")
> & "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" @params2
> ```
> 
> What does the custom timeout do, and why do you recommend it (I thought the timeout value is a parameter that you can use with Auto Dark Mode to specify how long you want to switch your theme for.)? Why should I use it? And do you think should I use force-dark or regular dark CLI in ADM?

The custom timeout specifies how long the shell waits for the ADM service to respond. It's to ensure compatibility as mentioned before, because some older versions don't support the ```--and-launch-service``` parameter without also specifying the timeout. It's because of the order in which cli arguments were parsed.

In your case it doesn't really matter if you use ```--force-light``` or ```--light``` and ```--force-dark``` or ```--dark``` since you exit the ADM service afterwards. Setting the force flag will bypass ADM's internal postpone mechanism and theme state checks. It will also prevent ADM from switching again on its own. Clearing the force status only matters if you want to keep the service alive, so I would recommend making use of the force flags.

**Comment by bigplayer-ai** [comment]
> In your case it doesn't really matter if you use --force-light or --light and --force-dark or --dark since you exit the ADM service afterwards. Setting the force flag will bypass ADM's internal postpone mechanism and theme state checks. It will also prevent ADM from switching again on its own. Clearing the force status only matters if you want to keep the service alive, so I would recommend making use of the force flags.

Actually it does differ. If you force night mode while using non forced dark mode. The theme won't switch if you set it for different time.
Can you check my last version of ADM specific code snippet?
if ($nightMode -eq $true)
{
    $params = @("--force-dark", "10", "--and-launch-service")
}
else 
{
    $params = @("dark", "10", "--and-launch-service")
}
& "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" @params
$params2 = @("--exit")
& "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" @params2

**Comment by Spiritreader** [comment]
> > In your case it doesn't really matter if you use --force-light or --light and --force-dark or --dark since you exit the ADM service afterwards. Setting the force flag will bypass ADM's internal postpone mechanism and theme state checks. It will also prevent ADM from switching again on its own. Clearing the force status only matters if you want to keep the service alive, so I would recommend making use of the force flags.
> 
> Actually it does differ. If you force night mode while using non forced dark mode. The theme won't switch if you set it for different time. Can you check my last version of ADM specific code snippet?
> 
> ```powershell
> # Use a ternary operator to assign different values to $params based on $nightMode
> $params = $nightMode ? @("--force-dark", "10", "--and-launch-service") : @("dark", "10", "--and-launch-service")
> 
> # Invoke AutoDarkModeShell.exe with $params
> & "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" @params
> 
> # Invoke AutoDarkModeShell.exe with --exit
> & "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" --exit
> ```



> 

No. If you have set up everything correctly, it doesn't matter if you use the force method or not. Please ensure that automatic switching is disabled in ADM and that your theme is configured correctly. 

Your snippet only generates ```--force-dark``` and ```--dark``` as parameters, that doesn't seem right.

I'm not involved with WDD. I merely reached out as courtesy for people that want to get theme switching to work alongside it without actually having to use the ADM background service. Which is technically not how the program is supposed to be used. 

My main focus lies with maintaining ADM. 
What you're doing currently should eventually lead to success, might require a bit of tinkering. 

Good luck!

**Comment by bigplayer-ai** [comment]
Ok thanks, It seems working pretty well atm. Also, It helped me with my mouse scheme (less clutter).

**Comment by Spiritreader** [comment]
> Ok thanks, It seems working pretty well atm. Also, It helped me with my mouse scheme (less clutter). I just noticed that you are part of auto dark mode program. Thank you for this amazing utility.

Sure thing, glad you like it!
Just wanted to clarify that I don't really know much about WDD and powershell, as such I can't really help out there.

**Comment by t1m0thyj** [comment]
> Honestly maintaining a functioning theme switch is nightmare territory. It almost never works reliably and you're chasing behavior that often breaks the entire windows UI for no apparent reason.
> 
> I would assume maintaining WDD by itself is already enough work as is (and so is ADM as a matter of fact)

Thanks for developing ADM ❤️ You are right, it's kind of a nightmare to maintain some features that are not approved by Microsoft 🙂 Setting a wallpaper across all virtual desktops is similar because it keeps breaking with new versions of Win11.

> It's already possible to call ADM via CLI, perform the operations you desire and then shut it down again, so you get no background app. You can also hide the tray icon so you won't even know its running temporarily. Yes you have to configure ADM a bit, but it should work.

This is awesome for folks who want to use WinDD and ADM together. Would you approve of having a PowerShell script in the [WDD-scripts](https://github.com/t1m0thyj/WDD-scripts) repo that would invoke the ADM CLI to change theme? If so I can add one along with some documentation for how to set it up.

**Comment by Spiritreader** [comment]
> @Spiritreader Thanks for developing ADM ❤️ You are right, it's kind of a nightmare to maintain some features that are not approved by Microsoft 🙂 Setting a wallpaper across all virtual desktops is similar because it keeps breaking with new versions of Win11.

@t1m0thyj Absoltuely. I've been loosely following the whole virtual desktop fiasco and every time a new Windows build releases it's an "oh no" scenario where the question ops up what broke this time because they changed the import GUIDs again.
And thanks for developing and maintaining WDD. Over the years we've been pointing users to your repo whenever we noticed they wanted fancier wallpaper functionality.

> This is awesome for folks who want to use WinDD and ADM together.

The easiest way still is to just run both apps simultaneously, we've made sure to include options such that other wallpaper tools don't cause collisions with ADM's theming engine.

>  Would you approve of having a PowerShell script in the [WDD-scripts](https://github.com/t1m0thyj/WDD-scripts) repo that would invoke the ADM CLI to change theme? If so I can add one along with some documentation for how to set it up.

Of course, go ahead! If you need more specific knowledge about the CLI than [our doc ](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/wiki/Command-line-arguments-%7C-parameters) on it provides, feel free to reach out.

In case you would like a more tight integration, it's possible to send the CLI messages to ADM directly via a namedpipe c# API that we provide.

https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/blob/master/AutoDarkModeComms/PipeClient.cs
https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/blob/master/AutoDarkModeComms/IMessageClient.cs
https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/blob/master/AutoDarkModeSvc/Communication/Command.cs

```C#
class MessageHandler
{
    public static IMessageClient Client { get; } = new PipeClient();
}
```

```C#
ApiResponse result = ApiResponse.FromString(MessageHandler.Client.SendMessageAndGetReply(Command.Dark));
```

**Comment by meddouribadis** [comment]
Hello everyone and thank you for your messages.
Here is the corrected bigplayer-ai script which seems to work at present.

```powershell
param (
    [Parameter(Mandatory=$true)][int]$daySegment2,  # 0 = Day, 1 = Night
    [Parameter(Mandatory=$true)][int]$daySegment4,  # -1 = N/A, 0 = Sunrise, 1 = Day, 2 = Sunset, 3 = Night
    [Parameter(Mandatory=$true)][bool]$nightMode,   # True if night mode is enabled
    [Parameter(Mandatory=$false)][string]$imagePath # Path to current wallpaper image
)

# Use a ternary operator to assign different values to $params based on $nightMode
$params = $nightMode ? @("--force-dark", "10", "--and-launch-service") : @("--force-light", "10", "--and-launch-service")

# Invoke AutoDarkModeShell.exe with $params
& "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" @params

# Invoke AutoDarkModeShell.exe with --exit
& "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" --exit
```

**Comment by adoginspace** [comment]
Works perfectly on my ARM64 device but throws the following error on my desktop:

    Error(s) running PowerShell script
    'scripts\ChangeWindowsTheme.ps1':
    
    At
    C:\Users\...\AppData\local\Programs\WinDynamicDesktop\scripts\ChangeWindowsTheme.ps1:9 char:22
    + $params = $nightMode ? @("--force-dark", "10", "--and-launch-service" ...
    +            ~
    Unexpected token '?' in expression or statement.
    + Categorylnfo : ParserError: (:) [], ParseException
    + FullyQualifiedErrorld : UnexpectedToken

Tried reinstalling both apps but no luck.

**Comment by Spiritreader** [comment]
> Works perfectly on my ARM64 device but throws the following error on my desktop:
> 
> ```
> Error(s) running PowerShell script
> 'scripts\ChangeWindowsTheme.ps1':
> 
> At
> C:\Users\...\AppData\local\Programs\WinDynamicDesktop\scripts\ChangeWindowsTheme.ps1:9 char:22
> + $params = $nightMode ? @("--force-dark", "10", "--and-launch-service" ...
> +            ~
> Unexpected token '?' in expression or statement.
> + Categorylnfo : ParserError: (:) [], ParseException
> + FullyQualifiedErrorld : UnexpectedToken
> ```
> 
> Tried reinstalling both apps but no luck.

Looks like your desktop doesn't have Powershell 7 or newer, the ternary operator didn't exist before that. in PS. Maybe that could be it.

**Comment by meddouribadis** [comment]
> Works perfectly on my ARM64 device but throws the following error on my desktop:
> 
> ```
> Error(s) running PowerShell script
> 'scripts\ChangeWindowsTheme.ps1':
> 
> At
> C:\Users\...\AppData\local\Programs\WinDynamicDesktop\scripts\ChangeWindowsTheme.ps1:9 char:22
> + $params = $nightMode ? @("--force-dark", "10", "--and-launch-service" ...
> +            ~
> Unexpected token '?' in expression or statement.
> + Categorylnfo : ParserError: (:) [], ParseException
> + FullyQualifiedErrorld : UnexpectedToken
> ```
> 
> Tried reinstalling both apps but no luck.

Hello, you can use this script : 

```ps
param (
    [Parameter(Mandatory = $true)][int]$daySegment2, # 0 = Day, 1 = Night
    [Parameter(Mandatory = $true)][int]$daySegment4, # -1 = N/A, 0 = Sunrise, 1 = Day, 2 = Sunset, 3 = Night
    [Parameter(Mandatory = $true)][bool]$nightMode, # True if night mode is enabled
    [Parameter(Mandatory = $false)][string]$imagePath # Path to current wallpaper image
)

$dark = If ($daySegment2 -eq 1) { "--force-dark" } Else { "--force-light" }

& "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" $dark 10 --and-launch-service

& "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" --exit
```

**Comment by adoginspace** [comment]
Thank you! that worked!

**Comment by freecodecampster** [comment]
TLDR; Windows 11 Settings>Personalisation>Colours>Choose your mode> Choose custom here rather than just light or dark for both seems to have fixed the inconsistent updating of system colours for me.

I'm using the powershell scripts with the desktop version to switch between light/dark mode as well as update the wallpaper. I noticed app theme would update but not always the system theme, taskbar would still be old colour mode. After killing process explorer.exe and starting again the system theme would update. Not an ideal solution. Seeing as both app and system themes are updated with two separate commands I thought I'd try setting colour mode to custom just in case there is a state bug with using powershell commands. Seems to have worked so far it's been two days and system colour theme is updating. Not sure if this is exactly related to OP or subsequent thread but thought I'd shave a workaround for a similar problem I was having.

**Comment by adoginspace** [comment]
Getting this problem again with the new update, even after changing the personalization options to custom. Does anyone know how to get the old AutoDarkMode script working for the new engine?

**Comment by t1m0thyj** [comment]
Here's an updated version of the script in https://github.com/t1m0thyj/WinDynamicDesktop/issues/427#issuecomment-1560019953:
```powershell
$params = $Input | ConvertFrom-Json

$dark = If ($params.daySegment2 -eq 1) { "--force-dark" } Else { "--force-light" }

& "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" $dark 10 --and-launch-service

& "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" --exit
```

**Comment by adoginspace** [comment]
Thank you! This worked perfectly.

**Comment by bigplayer-ai** [comment]
Here is an updated AutoDarkMode script that accounts for manual dark/light mode settings. Ensure that WDD runs in admin mode, which can be set up using Task Scheduler at login.

```ps1
# Convert the input from JSON
$params = $Input | ConvertFrom-Json

# Define the $dark variable based on the themeMode and daySegment2
$dark = switch ($params.themeMode) {
    0 {  # Automatic mode
        if ($params.daySegment2 -eq 1) { "--force-dark" } else { "--force-light" }
    }
    1 { "--force-light" }  # Light mode
    2 { "--force-dark" }   # Dark mode
    default { "--force-light" }  # Default to light mode if anything else
}

# Launch AutoDarkModeShell with the determined mode
& "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" $dark --and-launch-service

# Add a delay before exiting (e.g., 10 seconds)
Start-Sleep -Seconds 10

# Exit the AutoDarkModeShell
& "C:\Program Files (x86)\AutoDarkMode\AutoDarkModeShell.exe" --exit

```

**Comment by sannik-bej** [comment]
> Getting this problem again with the new update, even after changing the personalization options to custom. Does anyone know how to get the old AutoDarkMode script working for the new engine?

The method (that currently works):
Set the Accent Color to Automatic.

- WDD Version: 5.6 [Both System and App Theme scripts installed]
- OS Version: 11 24H2 (build 26100.2605)

**Comment by leostudiooo** [comment]
My personal solution is to run the auto dark mode app separately and disable the WDD script.

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Sannik Bej ***@***.***>
发送时间: Wednesday, January 8, 2025 12:42:34 AM
收件人: t1m0thyj/WinDynamicDesktop ***@***.***>
抄送: Leo Li ***@***.***>; Mention ***@***.***>
主题: Re: [t1m0thyj/WinDynamicDesktop] windows/app theme does not referesh the system after switching! (Issue #427)


Getting this problem again with the new update, even after changing the personalization options to custom. Does anyone know how to get the old AutoDarkMode script working for the new engine?

The method (that currently works):
Set the Accent Color to Automatic.

  *   WDD Version: 5.6 [Both System and App Theme scripts installed]
  *   OS Version: 11 24H2 (build 26100.2605)

―

**Comment by t1m0thyj** [comment]
This should be fixed now by https://github.com/t1m0thyj/WDD-scripts/pull/54 - thanks to @KrakenByte27

Please try downloading the latest versions of the scripts for [App Theme](https://github.com/t1m0thyj/WDD-scripts/blob/master/stable/ChangeWindowsAppTheme.ps1) and [System Theme](https://github.com/t1m0thyj/WDD-scripts/blob/master/stable/ChangeWindowsSystemTheme.ps1) and post back your results 🙂
